.github/workflows/pr_approval.yml

```diff
@@ -73,7 +74,6 @@ jobs:
               const tomlContent = fs.readFileSync('.github/pull_requests.toml', 'utf8');
               console.log('TOML content:', tomlContent);
               const tomlData = toml.parse(tomlContent);
-              console.log('Parsed TOML data:', JSON.stringify(tomlData, null, 2));
```

Comment:
Have you considered the business logic to a script instead of hard coding it in here? I think it is usually much easier to debug and run it locally. For example, you could create a script that takes as an argument a list of logins and the script succeeds if and only if the minimum number of people in the committee has been achieved.

Or the script output the number of people from the committee from that list and you only check here if the number is >= minimum requirement.

Reply:
Thank you for that idea! I shall move the logic to its own script ad use gh command line to try and rerun old failing checks.